.buildkite/scripts/test-on-platform.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+
+set -euo pipefail
+
+go version
+make lint
```

Comment:
I see that both in Jenkins and Buldkite lint is showing some differences but it does not fail the step/stage in CI.
https://beats-ci.elastic.co/job/Library/job/go-licenser-mbp/job/PR-64/9/console


Should this make fail the output ? 

Maybe adding this to the `lint` target in the Makefile?
```diff
 $ git diff
diff --git Makefile Makefile
index 4443308..2e946ad 100644
--- Makefile
+++ Makefile
@@ -61,6 +61,7 @@ install:
 lint: build
        @ go run honnef.co/go/tools/cmd/staticcheck@$(VERSION_STATICCHECK)
        @ gofmt -d -e -s .
+       @ test -z $(shell gofmt -l main.go | tee /dev/stderr)
        @ $(GOBIN)/go-licenser -d -exclude golden
 
 .PHONY: format
```

If this change is introduced, some files would needed to be updated (main.go and main_test.go)

Reply:
As discussed over slack, we should keep that for a later PR. The linting step shouldn't be a blocker for now.